Calibration/IsolatedParticles/plugins/IsolatedTracksCone.cc

```diff
@@ -334,12 +614,12 @@ void IsolatedTracksCone::analyze(const edm::Event& iEvent,
       nFailHighPurityQaul++;
     }
     
-    hRawPt ->Fill(pt1 );
-    hRawP  ->Fill(p1  );
-    hRawEta->Fill(eta1);
-    hRawPhi->Fill(phi1);
+    h_RawPt ->Fill(pt1);
+    h_RawP  ->Fill(p1 );
```

Comment:
minor comment, but since you are anyway making a cosmetic change here, you could also remove the remaining whitespace in `(p1 )`